libs/Onboarding/tests/qml_tests/tst_OnboardingController.qml

```diff
@@ -0,0 +1,86 @@
+import QtQuick
+import QtQml
+import QtTest
+
+import Status.Onboarding
+
+import Status.TestHelpers
+
+/**!
+ * \todo use mocked values
+ */
+Item {
+    id: root
+    width: 400
+    height: 300
+
+    Component {
+        id: onboardingDepsComponent
+
+        Item {
+            OnboardingModule {
+                id: module
+
+                userDataPath: "/tmp/StatusTests/demo"
+            }
+
+            // TODO: fix error "unable to assign Status::Onboarding::OnboardingController to Status::Onboarding::OnboardingController" then enable typed properties
+            property var /*OnboardingController*/ controller: module.controller
```

Comment:
`var` is the standard way of using custom C++ models/controllers in QML

Reply:
I'm not sure I understand why not to use typed properties as I was successfully doing in the Qt5 imperative registration approach? I expect this to work and will look into it after the POC.